scripts/create_test_subset.py

```diff
@@ -137,7 +172,19 @@ def main(
 
     else:
         assert samples_n
-        samples = random.sample(all_samples, samples_n)
+        if _additional_fams:
+            _additional_samples.extend(
+                get_samples_for_families(project, _additional_fams)
+            )
+
+        samples_to_subset = [
+            sample for sample in all_samples if sample['id'] not in _additional_samples
```

Comment:
Same O(n) lookup comment with `_additional_samples`, but more so as _additional_samples could be significantly larger.